scripts/runIchorCNA.R

```diff
@@ -58,11 +58,50 @@ option_list <- list(
   make_option(c("--txnStrength"), type="numeric", default=1e7, help = "Transition pseudo-counts. Exponent should be the same as the number of decimal places of --txnE. Default: [%default]"),
   make_option(c("--multSampleTxnStrength"), type="numeric", default=1, help="Strength of same state transition between multiple samples. Default: [%default]"),
   make_option(c("--plotFileType"), type="character", default="pdf", help = "File format for output plots. Default: [%default]"),
-	make_option(c("--plotYLim"), type="character", default="c(-2,2)", help = "ylim to use for chromosome plots. Default: [%default]"),
+  make_option(c("--plotYLim"), type="character", default="c(-2,2)", help = "ylim to use for chromosome plots. Default: [%default]"),
   make_option(c("--outDir"), type="character", default="./", help = "Output Directory. Default: [%default]"),
   make_option(c("--libdir"), type = "character", default=NULL, help = "Script library path. Usually exclude this argument unless custom modifications have been made to the ichorCNA R package code and the user would like to source those R files. Default: [%default]"),
   make_option(c("--cores"), type="numeric", default = 1, help = "Number of cores to use for EM. Default: [%default]")
 )
+
+# Function to parse character vectors
+parse_numeric_vector <- function(str_value) {
+  if (is.null(str_value) || str_value == "NULL" || str_value == "None") {
+    return(NULL)
+  }
+  # Remove "c(" and ")" if present
+  str_value <- gsub("^c\\(", "", str_value)
+  str_value <- gsub("\\)$", "", str_value)
```

Comment:
What happens if the string is not properly formatted? This is an example of what I wrote above, invalid input will create just more errors.